src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -360,10 +362,13 @@ private void onUserActivity() {
 
   public void onMouseEnteredClanTag() {
     clanService.getClanByTag(player.getClan()).thenAccept(optionalClan -> {
-      if (!optionalClan.isPresent() || clanMenu.getTooltip() != null) {
+      if (!optionalClan.isPresent()) {
         return;
       }
-      clanMenu.getItems().clear();
+
+      if (!clanMenu.getItems().isEmpty()) {
+        clanMenu.getItems().clear();
+      }
```

Comment:
Why do we only call `clear()` if it's not empty? Clearing an empty list shouldn't have any effect.